testsuite/replay_verify.py

```diff
@@ -67,7 +68,8 @@
     [321_000_001, 331_000_000],
     [331_000_001, 354_000_000],
     [354_000_001, 400_000_000],
-    [400_000_001, sys.maxsize],
+    [400_000_001, 490_000_000],
+    [490_000_000, sys.maxsize],
```

Comment:
<img width="206" alt="image" src="https://github.com/aptos-labs/aptos-core/assets/7528420/250b4950-5e16-46a8-8653-60bb450f533a">

Adding another job to the pool is sufficient to unblock the team, but it doesn't balance the load for overall latency. The time to get validation signal is determined by the slowest one in the pool. 

Can you re-balance the ranges in a subsequent PR?  For example, job(12) works on [295_000_001, 321_000_000] and takes 4hrs; 
job(15) works on [354_000_001, 400_000_000] and takes ~6hrs.

Reply:
will do as a follow up